code/modules/chemistry/Reagents-Drugs.dm

```diff
@@ -490,6 +490,86 @@ datum
 				..()
 				return
 
+		drug/solipsizine
+			name = "solipsizine"
+			id = "solipsizine"
+			description = "A highly potent hallucinogenic substance that causes intense delirium and acute inability to percieve others."
+			reagent_state = LIQUID
+			depletion_rate = 0.1
```

Comment:
Seems rather low

Reply:
i was originally torn between 0.1 and 0.2 depletion, but since it doesn't hinder you in any phisical way(you can still call out in the radio or simply go to the nearest source of charcoal) and that the effects take a minimum of 30 seconds to take place i found it fair for a full syringe of it to last 5 minutes, but i could still go back to the 0.2 depletion if you think that would be better.

Reply:
I think 0.2 would be better.

Reply:
can do